src/app/clusters/door-lock-server/door-lock-server-user.cpp

```diff
@@ -158,7 +158,7 @@ static void printUserTables(void)
 }
 
 // Returns status byte for use in SetPinResponse and SetRfidResponse commands.
-static uint8_t setUser(uint16_t userId, uint8_t userStatus, uint8_t userType, uint8_t * code,
+static uint8_t setUser(uint16_t userId, uint8_t userStatus, uint8_t userType, chip::ByteSpan code,
```

Comment:
```suggestion
static uint8_t setUser(uint16_t userId, uint8_t userStatus, uint8_t userType, ByteSpan code,
```
and same elsewhere in this file...